src/public_key_binary.rs

```diff
@@ -129,6 +129,12 @@ mod sqlx_postgres {
         }
     }
 
+    impl PgHasArrayType for PublicKeyBinary {
+        fn array_type_info() -> PgTypeInfo {
+            PgTypeInfo::with_name("_text")
+        }
```

Comment:
So the iot verifier has been using byte arrays in Postgres vs the text version that these Pg trait implementations have declared right?

Reply:
doesn't look like it. there is one instance of the iot_verifier saving _just_ a pubkeybin (in the gateway_shares table) where it's saving the key as `text`. all of the uses of `bytea` are storing more data than just a pubkey, such as a report which may _contain_ a pubkeybin in one of its fields, but the whole report struct is then serialized to bytes before being stored

i believe whenever we're storing _just_ a pubkeybin (formerly pubkey) throughout the oracles project it's being done as text

Reply:
Thanks for verifying